backend/src/interfaces/IConfig.ts

```diff
@@ -5,12 +5,3 @@ export interface IApiConfig {
 export interface IConfig {
```

Comment:
we can probably delete this entire file right?

Reply:
good point, I'll delete it right away!